consistent-sampling/src/main/java/io/opentelemetry/contrib/samplers/ConsistentComposedAndSampler.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.samplers;
+
+import static java.util.Objects.requireNonNull;
+
+import io.opentelemetry.contrib.state.OtelTraceState;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * A consistent sampler composes two consistent samplers.
+ *
+ * <p>This sampler samples if both samplers would sample.
```

Comment:
Can you describe the difference in behavior vs composing the actual `SamplingResult` instead?

Reply:
please see added javadoc in b3459f9e79d1a69c17e4cdf7aeb811801b692b6b